javascript_ts/11.qnamaker/README.MD

```diff
@@ -0,0 +1,58 @@
+This sample shows how to create a bot that uses QnA Maker with Application Insights to get telemetry around bot and QnA Maker performance. This bot example uses [`applicationinsights`](https://www.npmjs.com/package/applicationinsights), [`restify`](https://www.npmjs.com/package/restify) and [`dotenv`](https://npmjs.com/package/dotenv). 
+
+# Concepts introduced in this sample
+The [QnA Maker Service](https://www.qnamaker.ai) enables you to build, train and publish a simple question and answer bot based on FAQ URLs, structured documents or editorial content in minutes.
+
+The [Application Insights](https://azure.microsoft.com/en-us/services/application-insights/) enables you to discover actionable insights through application performance management and instant analytics.
+
+In this sample, we demonstrate how to use the QnA Maker service to answer questions based on a FAQ text file as input.
+
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-js.git
+    ```
+- In a terminal, navigate to javascript_ts\11.qnamaker
+    ```bash
+    cd javascript_ts\11.qnamaker
+    ```
+- Point to the myget feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+# Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
```

Comment:
replace with the aka link. [https://aka.ms/botframeworkemulator](https://aka.ms/botframeworkemulator)

Reply:
https://aka.ms/botframeworkemulator points to the last v3 emulator release while https://aka.ms/botframework-emulator points to the last stable preview v4 release. #Resolved